.gitignore

```diff
@@ -1,17 +1,18 @@
-node_modules
-bower_components
-public
-.DS_Store
-*.log
-js/tmpl/
-dist
-devel
-.polis_env
-.polis_s3_creds_client.json
-.polis_scp_creds_client.json
-.polis_slack_creds.json
-polis.config.js
-.sass-cache/*
-.bashrc
-.env_always
-polisStatic/
+node_modules
+bower_components
+public
+.DS_Store
+*.log
+js/tmpl/
+dist
+devel
+.polis_env
+.polis_s3_creds_client.json
+.polis_scp_creds_client.json
+.polis_slack_creds.json
+polis.config.js
+.sass-cache/*
+.bashrc
+.env_always
+polisStatic/
+.idea

```

Comment:
@urakagi Do you know why this file shows up as a complete rewrite when there was only one addition? I noticed this in another file, and it makes review a bit challenging.

Reply:
I have not mentioned this. My guess is that because I work on several (four) computers, there may be some sync issues for git. Because it's just `.gitignore`, so I overwrite it every time.

Reply:
in my experience, some of the mainline dockerfiles have windows line-endings. Correcting them to linux line-endings looks like a rewrite.

(Use `?w=1` in commit/compare urls to ignore whitespace)